### PR TITLE
Update sample systemd for container

### PR DESCRIPTION
**Pull request by SvenDowideit**
This replaces  #10653 

@dlintw thankyou for the info :)

### CODE REVIEW

**Comment by SvenDowideit** [comment]
@fredlf @jamtur01

**Comment by SvenDowideit** [comment]
@jfrazelle jenkins is sad?

**Comment by jfrazelle** [comment]
I restarted it, but just fyi, if you noticed on irc in docker-maintainers you can restart a build with `!rebuild 10723` ;)

**Comment by SvenDowideit** [comment]
LGTM - @fredlf @jamtur01 @spf13 @moxiegirl

File: docs/sources/articles/host_integration.md [hunk 1]
```diff
@@ -74,3 +73,14 @@ a new service that will be started after the docker daemon service has started.
 
     [Install]
     WantedBy=local.target
+
+If you need to pass options to the redis container (such as '--env'),
```
Comment (jamtur01) [inline docs/sources/articles/host_integration.md]: All of your formatting needs to be corrected - \`like this\`

Comment (jamtur01) [inline docs/sources/articles/host_integration.md]: Redis

**Comment by jfrazelle** [comment]
LGTM

**Pull request merged**